tests/integration/targets/connection_proxmox_pct_remote/dependencies.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Copyright (c) 2025 Nils Stein (@mietzen) <github.nstein@mailbox.org>
+# Copyright (c) 2025 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- hosts: localhost
+  gather_facts: true
+  serial: 1
+  tasks:
+    - name: Define pct mock destination
+      set_fact:
+        pct_dest: >-
+          {{ "/opt/homebrew/sbin/pct" if ansible_distribution == "MacOSX"
```

Comment:
Why not
```suggestion
          {{ "/opt/homebrew/bin/pct" if ansible_distribution == "MacOSX"
```
?

Reply:
I got `/usr/sbin` hard coded, it need to go there. But it is write protected. I'm trying to remount / `mount -uw /`

Reply:
You won't be able to write to /usr/sbin on macOS, no matter how hard you try.

If you really need /usr/sbin, then maybe add `skip/macos` back :)

Reply:
Is docker installed on macOS and is it even possible on freeBSD? Then we could build a Docker image with the pct mock and a openssh server running on e.g. port 2222 and connect to that.

Reply:
There is no Docker on macOS, and it's not possible to install it on FreeBSD. I'd simply skip macOS.